src/main/java/com/openlattice/shuttle/PropertyDefinition.java

```diff
@@ -185,7 +185,7 @@ public Builder(
             return this;
         }
 
-        public Builder<T> value( List<String> columns, String hashFunction ) {
+        public Builder<T> value( List<String> columns, HashTransform.HashType hashFunction ) {
```

Comment:
@katherinebernstein When changing the hash function to an enum, I had to change this.  I think you're the only one using the builder at this point.  Make sure this won't affect your ongoing integrations.